### ISSUE TITLE
training

### ISSUE THREAD

**Issue by zezeeze** [MahmudulAlam/Automatic-Identification-and-Counting-of-Blood-Cells#22]
The loss during model training is 10-20, how to solve it? Can YOLOv7 or higher versions be used?

**Comment by MahmudulAlam** [comment]
@zezeeze 

The value of the loss can be subjective. What is your accuracy rate?

**Comment by zezeeze** [comment]
The accuracy rate is more than 85%, and the effect is good. So although the loss is high, it does not affect? Another problem is the checkpoint,.index,.meta,.profile and.data-00000-of-00001 files generated in the weights folder after my training. How do I continue training on the basis of the new data? I sought the help of chatgpt and related students, but did not solve the problem. Thank you.

**Comment by MahmudulAlam** [comment]
@zezeeze  The size of the loss value sometimes depends on the definition. If you define by loss = sum(error), the number would be higher compared to loss = mean(error). If you want to improve performance, use more data, apply data augmentation, etc. I don't think loss value would help you here. 

To resume/continue training use the checkpoint number after --load checkpoint_num 
```
python flow --model cfg/tiny-yolo-voc-3c.cfg --load 1500 --train --gpu .7 --annotation dataset/Training/Annotations --dataset dataset/Training/Images --lr 1e-3 --epoch 100
```

**Comment by zezeeze** [comment]
真的非常感谢你的帮助！！！！我按照你说的方法已经解决了。

**Comment by zezeeze** [comment]
Thank you so much for your help ！！！！ I worked it out the way you said.
